pkg/runit/servicebuilder.go

```diff
@@ -113,6 +117,19 @@ __END__
 	return []byte(ret), nil
 }
 
+func (s ServiceTemplate) finishScript() ([]byte, error) {
+	finish_exec := s.Finish
+	if len(finish_exec) == 0 {
+		finish_exec = []string{"/bin/true", "# finish not implemented"}
```

Comment:
The template is evaluated into a file on the disk that will be invoked by runit. I don't see the connection to the `os` package.